src/openrct2/world/Sprite.cpp

```diff
@@ -28,6 +28,7 @@
 
 static rct_sprite _spriteList[MAX_SPRITES];
 static std::array<std::list<uint16_t>, EnumValue(EntityListId::Count)> gEntityLists;
+static std::vector<uint16_t> _freeIdList;
```

Comment:
I took a very quick look and I don't understand why this has to be a vector to count the number of free entities 🤔

Reply:
Its not the count, it holds the index of a free entity.

Reply:
Hmm, I see. Thanks!